app/src/main/java/org/thoughtcrime/securesms/database/MmsDatabase.kt

```diff
@@ -276,6 +276,16 @@ class MmsDatabase(context: Context, databaseHelper: SQLCipherOpenHelper) : Messa
         notifyConversationListeners(threadId)
     }
 
+    fun markAsSyncing(messageId: Long) {
```

Comment:
Hmm, I wonder if we could bring these, or the `markAs` functionality to the `MessagingDatabase` to save doing it in both the Mms and Sms dbs?